src/test/scala/com/advancedtelematic/campaigner/daemon/DeviceEventListenerSpec.scala

```diff
@@ -30,6 +32,12 @@ class DeviceEventListenerSpec extends CampaignerSpec with DatabaseSpec with Devi
     Messages.DeviceEventMessage(campaign.namespace, event)
   }
 
+  private def genEventOfType(eventType: EventType): DeviceEventMessage = {
```

Comment:
if we are using Generators and not returning a `Gen[_]` then the method name should not start with `gen`

Reply:
Sure. Same question about code style as in a comment about `yield` braces applies.